src/lib/ngsiNotify/senderThread.cpp

```diff
@@ -114,16 +133,37 @@ void* startSenderThread(void* p)
       {
         __sync_fetch_and_add(&noOfNotificationsSent, 1);
         alarmMgr.notificationErrorReset(url);
+        params->failsCounter = 0;
 
         if (params->registration == false)
         {
           subNotificationErrorStatus(params->tenant, params->subscriptionId, 0, statusCode, "");
         }
       }
+
+
       else
       {
         alarmMgr.notificationError(url, "notification failure for sender-thread: " + out);
 
+        params->failsCounter = params->failsCounter +1;
```

Comment:
Maybe better (more compact):

```suggestion
        params->failsCounter++;
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/0ebe8e05845a5d383baede531e80f33d57dced29